### ISSUE TITLE
Focus order is incorrect when navigating through the associated events in the date cell of the calendar in Windows.

### ISSUE THREAD

**Issue by msftedad** [jquense/react-big-calendar#2713]
### Check that this is really a bug

- [x] I confirm

### Reproduction link

https://jquense.github.io/react-big-calendar/examples/index.html?path=/story/about-big-calendar--page

### Bug description

Focus order is incorrect when navigating through the associated events in the date cell of the calendar in Windows. Keyboard users will find it difficult to navigate and get the information correctly if focus order is incorrect.

### Expected Behavior

On pressing tab key, focus order should be correct when navigating through the associated events in the date cell of the calendar. Focus should move to the events in the date cell for ex: after 7th date cell focus should move to the 'events in that date and then to the next date cell. similarly, focus should also move to the controls like '+4 more' and then move to the next date cell.

### Actual Behavior

On pressing tab key, focus order is incorrect when navigating through the associated events in the date cell of the calendar. Focus is not moving to the events in the date cell for ex: after 7th date cell focus is going to the next date cell.

<img width="969" alt="Image" src="https://github.com/user-attachments/assets/78e37f4d-e4ae-487e-82ed-724b55d44baa" />

https://github.com/user-attachments/assets/5c690086-c4f0-4c0f-86bf-b9a708bb3fc9

### react-big-calendar version

0.33.2

### React version

16.8.1

### Platform/Target and Browser Versions

Windows 11 Version 24H2(OS Build 26120.2415)

### Validations

- [x] Read the [docs](http://jquense.github.io/react-big-calendar/examples).
- [x] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [x] Make sure this is a react-big-calendar issue and not an implementation issue

### Would you like to open a PR for this bug?

- [x] I'm willing to open a PR

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
